### PR TITLE
Fix typos in README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SulaimanAminuBarkindo**
I fixed the typos I was able to find and also fixed some grammar and rephrase some wordings in the README.md and CONTRIBUTING.md. 
Please Review.

### CODE REVIEW

**Comment by ismaelsadeeq** [comment]
Hello  @Nakamcode Kindly Review this PR, If It is Okay you can merge

**Comment by Nakamcode** [comment]
Sure, I'm reviewing it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,69 +1,70 @@
...
+Be confident that your code works. When was the last time you committed a code change, your build failed and half of your app stopped working? Mine was last week. Writing tests for our Algorithms will help us ensure the implementations are air tight even after multiple fixes and code changes.
 
 It is advised that the algorithm file (module) does not contain any "live" code but rather just exports the function(s) needed to execute the algorithm.
 
 Please refrain from printing to the console in your implementation AND test code.
 
 
-Wriiten by @ismaelsadeeq 2022
+Written by @ismaelsadeeq 2022
+Edited by @SulaimanAminuBarkindo 2022
```
Comment (Nakamcode) [inline CONTRIBUTING.md:70]: This is not important here

Comment (SulaimanAminuBarkindo) [reply CONTRIBUTING.md:70]: > This is not important here

Yes I agree, but what is your suggestion please. 
Should I remove both the Written by and Edited by or leave the Written by

Comment (Nakamcode) [reply CONTRIBUTING.md:70]: Edited by;

Comment (SulaimanAminuBarkindo) [reply CONTRIBUTING.md:70]: Alright, I will remove the Edited by then

Comment (Nakamcode) [reply CONTRIBUTING.md:70]: Sure, thanks for your understanding

Comment (SulaimanAminuBarkindo) [reply CONTRIBUTING.md]: I effect the changes requested. Please review. 
Thank you very much for your time

Comment (ismaelsadeeq) [reply CONTRIBUTING.md]: Hello Man @Nakamcode  If the Review is completed we can resolve the conversation, merge the PR and close the issue.
Thanks

Comment (Nakamcode) [reply CONTRIBUTING.md]: yeah

**Comment by Nakamcode** [comment]
Looking Good, thanks for your contribution !!

**Pull request merged**